modules/sdk-coin-xrp/src/ripple.ts

```diff
@@ -64,12 +63,8 @@ const signWithPrivateKey = function (txHex, privateKey, options) {
   const serialized = binary.encode(tx);
   return {
     signedTransaction: serialized,
-    id: computeBinaryTransactionHash(serialized),
+    id: xrpl.hashes.hashSignedTx(serialized),
   };
 };
 
-export = (params): ripple.RippleAPI => {
-  const rippleLib = new ripple.RippleAPI(params);
-  (rippleLib as any).signWithPrivateKey = signWithPrivateKey;
-  return rippleLib;
-};
+export = { ...xrpl, signWithPrivateKey };
```

Comment:
Seems to be breaking change?, Can you kindly mention in the commit message ([format](https://www.conventionalcommits.org/en/v1.0.0/#:~:text=in%20Semantic%20Versioning\).-,BREAKING,-CHANGE%3A%20a))